test/presenters/breadcrumbs_test.rb

```diff
@@ -0,0 +1,67 @@
+require 'test_helper'
+
+class BreadcrumbsTest < ActionView::TestCase
+  def build_content_item(links = [])
+    ItemWithBreadrumbs.new(
+      "title" => "a title",
+      "description" => "a description",
+      "format" => "a format",
+      "phase" => "live",
+      "document_type" => "content_item",
+      "links" => links,
+    )
+  end
+
+  def test_returns_empty_breadcrumbs_when_parent_is_not_specified
+    content_item = build_content_item
+    assert_equal [], content_item.breadcrumbs
+  end
+
+  def test_returns_empty_breadcrumbs_when_parent_is_empty
+    links = { parent: [] }
+    content_item = build_content_item(links)
+    assert_equal [], content_item.breadcrumbs
+  end
+
+  def test_returns_breadcrumbs_when_there_is_a_parent
+    links = {
+      "parent" => [
+        { "title" => "A-parent", "base_path" => "/a-parent" }
+      ]
+    }
+
+    content_item = build_content_item(links)
+
+    assert_equal [
+      { title: "Home", url: "/" },
+      { title: "A-parent", url: "/a-parent" },
+    ], content_item.breadcrumbs
+  end
+
+  def test_returns_breadcrumbs_when_there_is_more_than_one_parent
+    links = {
+      "parent" => [
+        {
+          "title" => "A-parent",
+          "base_path" => "/a-parent",
+          "parent" => [{
+            "title" => "Another-parent",
+            "base_path" => "/another-parent",
+          }]
+        }
+      ]
+    }
+
+    content_item = build_content_item(links)
+
+    assert_equal [
+      { title: "Home", url: "/" },
+      { title: "Another-parent", url: "/another-parent" },
+      { title: "A-parent", url: "/a-parent" },
+    ], content_item.breadcrumbs
+  end
+end
+
+class ItemWithBreadrumbs < ContentItemPresenter
```

Comment:
can you move this to the top of this file, it's better to read that you created this class in this file before the tests